platform/lib/platform_web/components.ex

```diff
@@ -3228,6 +3228,14 @@ defmodule PlatformWeb.Components do
         if(attr.schema_field == :project_attributes, do: :value, else: attr.schema_field)
       )
 
+    # add coordinates here
+    {lat, lon} = assigns.media.attr_geolocation.coordinates
```

Comment:
`assigns.media` can be `nil` when creating a new incident. This would result in `(KeyError) key :attr_geolocation not found in: nil`